frontend/src/components/CreateCollectionModal/components/Content/components/LinkInput/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { IconNames } from "@blueprintjs/icons";
+import { debounce } from "lodash-es";
+import React, { FC } from "react";
+import Input from "src/components/common/Form/Input";
+import { GRAY } from "src/components/common/theme";
+import { DEBOUNCE_TIME_MS } from "src/components/CreateCollectionModal/components/Content/common/constants";
+import { IconWrapper, StyledButton, Wrapper } from "./style";
+
+export type LinkValue = {
+  id: number;
+  value: string;
+  isValid: boolean;
+  index: number;
+};
+
+export enum TYPES {
+  DOI = "DOI",
+  RAW_DATA = "RAW_DATA",
+  PROTOCOL = "PROTOCOL",
+  LAB_WEBSITE = "LAB_WEBSITE",
+  OTHER = "OTHER",
+}
+
+export const OPTIONS = {
+  [TYPES.DOI]: { text: "DOI", value: TYPES.DOI },
+  [TYPES.RAW_DATA]: { text: "Raw Data", value: TYPES.RAW_DATA },
+  [TYPES.PROTOCOL]: { text: "Protocol", value: TYPES.PROTOCOL },
+  [TYPES.LAB_WEBSITE]: { text: "Lab Website", value: TYPES.LAB_WEBSITE },
```

Comment:
Is this assigning the value of the enum keys to the keys of `OPTIONS`? Wild!

Reply:
Yup! Yeah pretty convenient lol